libyul/optimiser/StackLimitEvader.cpp

```diff
@@ -174,7 +174,7 @@ void StackLimitEvader::run(
 		"StackLimitEvader can only be run on objects using the EVMDialect with object access."
 	);
 
-	std::vector<FunctionCall*> memoryGuardCalls = FunctionCallFinder::run(
```

Comment:
Somehow I preferred the old way with the `FunctionCallFinder` struct.

Reply:
Why though? It seems perfectly natural to me. On the other hand having a static `run()` in a class that carries no state is less so.

Reply:
I guess it just looks cooler ;)